Documentation/contributing/development/season_of_docs.rst

```diff
@@ -10,11 +10,12 @@
 Google Season of Docs 2022
 ==========================
 
-Cilium is interested in applying to the 2022 Google Season of Docs to improve the documentation experience for our users and contributors.
-Below is our project ideas, if you have any questions, please reach out to gsod@cilium.io.
+Cilium has been accepted into the 2022 Google Season of Docs to improve the documentation experience for our users and contributors.
+Below is our project ideas, if you have any questions, please reach out to gsod@cilium.io. Technical writers can apply until April 27th
+in this `form`_.
 
 Reorganize Cilium Features and Getting Started 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+===================================
```

Comment:
The underline needs to be the same length as the title (careful if you edit online, the text box may not use monospace fonts).

Also with `=`, you raised this title to the same level as the page title. Can we do something intermediate? Any other symbol should do I think, e.g. dashes `----`. Sphinx should assign the title levels as it finds them, so it should automatically turn the ones below into `<h3>` instead of `<h2>`.

(Bonus: there's an additional space at the end of `Reorganize Cilium Features and Getting Started␣`, which you could remove.)

Reply:
ahh, was wondering why it was different lengths, I'll fix it now